Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -8,8 +8,13 @@ import QuartzCore
 class LayoutManager: NSLayoutManager {
 
     /// Blockquote's Left Border Color
-    ///
-    var blockquoteBorderColor: UIColor? = UIColor(red: 0.52, green: 0.65, blue: 0.73, alpha: 1.0)
+    /// Include 3 nested values
```

Comment:
This comment is outdated, probably should indicate what is the purpose of the variable and don't mention the number of levels that can be changed depending of the needs.